src/mono/wasm/debugger/BrowserDebugProxy/DevToolsHelper.cs

```diff
@@ -209,6 +209,10 @@ public static MonoCommands EvaluateMemberAccess(int scopeId, string expr, params
         public static MonoCommands Resume() => new MonoCommands($"MONO.mono_wasm_debugger_resume ()");
 
         public static MonoCommands SetPauseOnExceptions(string state) => new MonoCommands($"MONO.mono_wasm_set_pause_on_exceptions(\"{state}\")");
+
+        public static MonoCommands SetDebugJustMyCode(int state) => new MonoCommands($"MONO.mono_wasm_set_debug_just_my_code({state})");
+
+        public static MonoCommands AddUserAssembly(string assemblyName) => new MonoCommands($"MONO.mono_wasm_add_user_assembly(\"{assemblyName}\")");
```

Comment:
Should we really be calling this something like `mark_as_user_assembly`, instead of `*add*_user_assembly`?

Reply:
okay.